### PR TITLE
[docs] Move ORMs to Develop section

### LINKED ISSUE #15012: [Docs] Move ORMs from build an app to Drivers and ORMs
### Description

Create a section under Drivers and ORMs and move the examples here.

### PR DESCRIPTION
**Pull request by aishwarya24**
resolves #15012 
Build preview: https://deploy-preview-15122--infallible-bardeen-164bc9.netlify.app/preview/drivers-orms/orms/

### CODE REVIEW

File: docs/content/preview/drivers-orms/java/_index.md [hunk 1]
```diff
@@ -27,10 +27,10 @@ The following projects can be used to implement Java applications using the Yuga
...
-| Spring Data YugabyteDB | [Documentation](../../integrations/spring-framework/sdyb/)<br />[Hello World App](../../develop/build-apps/java/ysql-sdyb/)<br />[Blog](https://blog.yugabyte.com/spring-data-yugabytedb-getting-started/) | [Spring Data YugabyteDB Sample App](https://github.com/yugabyte/spring-data-yugabytedb-example/)
-| Spring Data JPA | [Documentation](../../integrations/spring-framework/sd-jpa/)<br />[Hello World App](../../develop/build-apps/java/ysql-spring-data/)<br />[Blog](https://blog.yugabyte.com/run-the-rest-version-of-spring-petclinic-with-angular-and-distributed-sql-on-gke/) | [Spring Data JPA App](https://github.com/yugabyte/orm-examples/tree/master/java/spring/)
+| Ebean ORM | [Documentation](ebean/)<br /> [Hello World](../orms/ysql-ebean/)<br /> [Blog](https://blog.yugabyte.com/ebean-orm-yugabytedb/)| [Ebean ORM App](https://github.com/yugabyte/orm-examples/tree/master/java/ebean/)
```
Comment (ddhodge) [inline docs/content/preview/drivers-orms/java/_index.md:30]: For Go, Hello Worlds are in Example Apps column

File: docs/content/preview/drivers-orms/java/_index.md [hunk 2]
```diff
@@ -27,10 +27,10 @@ The following projects can be used to implement Java applications using the Yuga
...
+| Hibernate ORM | [Documentation](hibernate/)<br />[Hello World](../orms/ysql-hibernate/)<br />[Blog](https://blog.yugabyte.com/run-the-rest-version-of-spring-petclinic-with-angular-and-distributed-sql-on-gke/)<br /> | [Hibernate ORM](https://github.com/yugabyte/orm-examples/tree/master/java/hibernate/)
+| Spring Data YugabyteDB | [Documentation](../../integrations/spring-framework/sdyb/)<br />[Hello World](../../develop/build-apps/java/ysql-sdyb/)<br />[Blog](https://blog.yugabyte.com/spring-data-yugabytedb-getting-started/) | [Spring Data YugabyteDB Sample App](https://github.com/yugabyte/spring-data-yugabytedb-example/)
```
Comment (ddhodge) [inline docs/content/preview/drivers-orms/java/_index.md:32]: Why is this hello world still in build apps?

File: docs/content/preview/drivers-orms/orms/_index.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+menu:
+  preview:
+    identifier: orm-tutorials
+    parent: drivers-orms
+    weight: 610
+type: indexpage
+showRightNav: true
+---
+
+The examples in this section illustrate how you can buil an application by implementing a REST API server using the ORMs of your choice with YugabyteDB.
```
Comment (ddhodge) [inline docs/content/preview/drivers-orms/orms/_index.md:16]:
```suggestion
The examples in this section illustrate how you can build a basic online e-commerce store application that connects to YugabyteDB by implementing a REST API server using the ORM of your choice.

```

File: docs/content/preview/drivers-orms/orms/_index.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+The source for the REST API based application can be found in the [Using ORMs with YugabyteDB](https://github.com/yugabyte/orm-examples/tree/master) repository. The database access is managed using the ORM and includes the following tables:
```
Comment (ddhodge) [inline docs/content/preview/drivers-orms/orms/_index.md:17]:
```suggestion
The source for the REST API-based applications is in the [Using ORMs with YugabyteDB](https://github.com/yugabyte/orm-examples/tree/master) repository. Database access is managed using the ORM and includes the following tables:
```

File: docs/content/preview/drivers-orms/orms/_index.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+
+- `users` — the users of the e-commerce site
+- `products` — the products being sold
+- `orders` — the orders placed by the users
+- `orderline` — each line item of an order
+
+## Choose your language
```
Comment (ddhodge) [inline docs/content/preview/drivers-orms/orms/_index.md:24]:
```suggestion
### Choose your language
```

File: docs/content/preview/drivers-orms/orms/ysql-dapper.md [hunk 6]
```diff
@@ -1,13 +1,13 @@
 ---
 title: Build a C# application that uses Dapper ORM and YSQL
 headerTitle: Build a C# application
```
Comment (ddhodge) [inline docs/content/preview/drivers-orms/orms/ysql-dapper.md:3]: Need better headings for the se pages - 'Build a X application' is already used for a different set of pages. Something like 'C# ORM example application'

File: docs/content/preview/drivers-orms/orms/ysql-dapper.md [hunk 7]
```diff
@@ -1,13 +1,13 @@
 ---
```
Comment (ddhodge) [inline docs/content/preview/drivers-orms/orms/ysql-dapper.md:1]: Something odd going on in the left nav with multiple items selected

Comments for this page basically apply to all of them

File: docs/content/preview/drivers-orms/orms/ysql-dapper.md [hunk 8]
```diff
@@ -26,20 +26,15 @@ type: docs
...
 
-The following tutorial implements a REST API server using the [Dapper](https://github.com/DapperLib/Dapper) ORM. The scenario is that of an e-commerce application where database access is managed using the ORM. It includes the following tables:
-
-- `users` — the users of the e-commerce site
-- `products` — the products being sold
-- `orders` — the orders placed by the users
-- `orderline` — each line item of an order
+The following tutorial implements a REST API server using the [Dapper](https://github.com/DapperLib/Dapper) ORM. The scenario is that of an e-commerce application where database access is managed using the ORM.
 
 The source for the above application can be found in the [Using ORMs with YugabyteDB](https://github.com/yugabyte/orm-examples/tree/master/csharp/dapper) repository.
```
Comment (ddhodge) [inline docs/content/preview/drivers-orms/orms/ysql-dapper.md:31]:
```suggestion
The source for the application can be found in the [Using ORMs with YugabyteDB](https://github.com/yugabyte/orm-examples/tree/master/csharp/dapper) repository.
```

File: docs/content/preview/drivers-orms/orms/_index.md [hunk 9]
```diff
@@ -0,0 +1,69 @@
+---
+title: ORM tutorials
+headerTitle: ORMs
```
Comment (ddhodge) [inline docs/content/preview/drivers-orms/orms/_index.md:3]: Need a better heading - ORM hello world? ORM e-commerce app? or somesuch

File: docs/content/preview/drivers-orms/orms/ysql-dapper.md [hunk 10]
```diff
@@ -26,20 +26,15 @@ type: docs
...
 
 ## Prerequisites
 
 This tutorial assumes that you have:
 
-- YugabyteDB up and running. Download and install YugabyteDB by following the steps in [Quick start](../../../../quick-start/).
+- YugabyteDB up and running. Download and install YugabyteDB by following the steps in [Quick start](../../../quick-start/).
 - [.NET SDK 6.0](https://dotnet.microsoft.com/en-us/download) or later.
 
 ## Clone the "orm-examples" repository
```
Comment (ddhodge) [inline docs/content/preview/drivers-orms/orms/ysql-dapper.md:40]:
```suggestion
### Clone the "orm-examples" repository
```

Comment (ddhodge) [inline docs/content/preview/drivers-orms/orms/ysql-dapper.md:40]: Database configuration -> Set up the database connection [all]

File: docs/content/preview/drivers-orms/orms/ysql-dapper.md (hunk 8 above)
Comment (ddhodge) [inline docs/content/preview/drivers-orms/orms/ysql-dapper.md:31]: Do we need 'Explore the source' section at the bottom, seems redundant

File: docs/content/preview/drivers-orms/orms/ysql-django.md [hunk 11]
```diff
@@ -26,39 +26,24 @@ type: docs
...
-This repository has a Django ORM example that implements a basic REST API server. Database access in this application is managed through the Django ORM. The e-commerce database `ysql_django` includes the following tables:
-
-- `users` stores users of the e-commerce site.
-- `products` contains a list of products the e-commerce site sells.
-- `orders` contains orders placed by the users.
-- `orderline` stores multiple line items from an order.
-
-The source for the above application can be found in the `python/django` directory of Yugabyte's [Using ORMs with YugabyteDB](https://github.com/yugabyte/orm-examples) repository.
-
 ## Set up the application
```
Comment (ddhodge) [inline docs/content/preview/drivers-orms/orms/ysql-django.md:47]: Fix the indentation of code blocks

File: docs/content/preview/drivers-orms/orms/ysql-ebean.md [hunk 12]
```diff
@@ -1,57 +1,46 @@
...
 The source for this application can be found in the [Using ORMs with YugabyteDB](https://github.com/yugabyte/orm-examples/tree/master/java/ebean) repository.
 
 ## Prerequisites
 
 This tutorial assumes that you have:
 
-- YugabyteDB up and running. If you are new to YugabyteDB, follow the steps in [Quick start](../../../../quick-start/) to have YugabyteDB up and running in minutes.
+- YugabyteDB up and running. If you are new to YugabyteDB, follow the steps in [Quick start](../../../quick-start/) to have YugabyteDB up and running in minutes.
 - Java Development Kit (JDK) 1.8 is installed. JDK installers for Linux and macOS can be downloaded from [OpenJDK](http://jdk.java.net/), [AdoptOpenJDK](https://adoptopenjdk.net/), or [Azul Systems](https://www.azul.com/downloads/zulu-community/). Homebrew users on macOS can install using `brew install AdoptOpenJDK/openjdk/adoptopenjdk8`.
 - [sbt](https://www.scala-sbt.org/1.x/docs/) is installed.
```
Comment (ddhodge) [inline docs/content/preview/drivers-orms/orms/ysql-ebean.md:46]: broken YSQL link below

File: docs/content/preview/drivers-orms/orms/ysql-pg.md [hunk 13]
```diff
@@ -1,13 +1,13 @@
 ---
```
Comment (ddhodge) [inline docs/content/preview/drivers-orms/orms/ysql-pg.md:1]: This one isn't really like the others. Does it belong here?

File: docs/content/preview/drivers-orms/orms/ysql-prisma.md [hunk 14]
```diff
@@ -98,7 +93,7 @@ DATABASE_URL="postgresql://<user>:<password>@<host>:<port>/<db_name>"
 
 If you have a YugabyteDB Managed cluster, do the following:
 
-1. Download your [cluster certificate](../../cloud-add-ip/#download-your-cluster-certificate).
+1. Download your [cluster certificate](/preview/yugabyte-cloud/cloud-secure-clusters/cloud-authentication/#download-your-cluster-certificate).
```
Comment (ddhodge) [inline docs/content/preview/drivers-orms/orms/ysql-prisma.md:96]: relative link

File: docs/content/preview/integrations/spring-framework/sd-jpa.md [hunk 15]
```diff
@@ -21,7 +21,7 @@ This document describes the fundamentals of using Spring Data JPA with YugabyteD
 
 ## Quick start
 
-Learn how to establish a connection to YugabyteDB database and begin basic CRUD operations using the steps in [Build an Application](../../../develop/build-apps/java/ysql-spring-data/) in the Quick Start section.
+Learn how to establish a connection to YugabyteDB database and begin basic CRUD operations using the steps in [Build an Application](../../../drivers-orms/orms/ysql-spring-data/).
```
Comment (ddhodge) [inline docs/content/preview/integrations/spring-framework/sd-jpa.md:24]: Change link title to whatever page ends up being called

**Review by ddhodge — COMMENTED** [review]
Should these REST ORM examples go under Real-world examples instead

File: docs/content/preview/drivers-orms/java/_index.md (hunk 2 above)
Comment (aishwarya24) [reply docs/content/preview/drivers-orms/java/_index.md:32]: Yes this is one I have mentioned in the description. This one is not an ORM. I need to combine it with its integrations page.

File: docs/content/preview/drivers-orms/orms/ysql-pg.md (hunk 13 above)
Comment (aishwarya24) [reply docs/content/preview/drivers-orms/orms/ysql-pg.md:1]: nice catch , yes this one is not a REST example.

File: docs/content/preview/drivers-orms/go/pg.md [hunk 16]
```diff
@@ -66,68 +75,288 @@ db := pg.Connect(opt)
...
+
+#### Use SSL
+
+You can choose to enable or disable SSL for your local YugabyteDB cluster. To configure SSL/TLS for your YugabyteDB cluster, refer to [Enable client-to-server encryption](../../../secure/tls-encryption/client-to-server/). YugabyteDB Managed requires SSL connections, and SSL/TLS is enabled by default for client-side authentication.
+
+#### CA certificate
+
+Use the [CA certificate](../../../secure/tls-encryption/server-certificates/#generate-the-root-certificate-file) generated above as part of the SSL/TLS configuration of your cluster.
+
+In case of a YugabyteDB Managed cluster, to download the CA certificate for your cluster in YugabyteDB Managed, do the following:
```
Comment (ddhodge) [inline docs/content/preview/drivers-orms/go/pg.md:93]: just link to the download certificate topic in YBM docs

File: docs/content/preview/drivers-orms/go/pg.md [hunk 17]
```diff
@@ -66,68 +75,288 @@ db := pg.Connect(opt)
...
+1. Click **Connect**.
+
+1. Click **Connect to your application** and download the CA cert.
+
+#### OpenSSL
+
+Install [OpenSSL](https://www.openssl.org/) 1.1.1 or later only if you have a YugabyteDB setup with SSL/TLS enabled. YugabyteDB Managed clusters are always SSL/TLS enabled.
 
-### Step 3: Create tables
+The following table summarizes the SSL modes and their support in the driver:
```
Comment (ddhodge) [inline docs/content/preview/drivers-orms/go/pg.md:105]: Move this stuff to under Use SSL; delete the table, just say the driver supports all SSL modes; and then the YBM bit

File: docs/content/preview/drivers-orms/go/pg.md [hunk 18]
```diff
@@ -66,68 +75,288 @@ db := pg.Connect(opt)
...
-| port |  Listen port for YSQL | 5433
-| dbname | database name | yugabyte
-| sslMode | SSL mode | require
+| user | User for connecting to the database | yugabyte |
+| password | User password | yugabyte |
+| host  | Hostname of the YugabyteDB instance | localhost |
+| port |  Listen port for YSQL | 5433 |
+| dbname | Database name | yugabyte |
+| sslMode | SSL mode | require |
+
```
Comment (ddhodge) [inline docs/content/preview/drivers-orms/go/pg.md:84]: what about the certificate? should ssl go under ssl section instead

File: docs/content/preview/drivers-orms/go/pg.md [hunk 19]
```diff
@@ -66,68 +75,288 @@ db := pg.Connect(opt)
...
+#### Use SSL
+
+You can choose to enable or disable SSL for your local YugabyteDB cluster. To configure SSL/TLS for your YugabyteDB cluster, refer to [Enable client-to-server encryption](../../../secure/tls-encryption/client-to-server/). YugabyteDB Managed requires SSL connections, and SSL/TLS is enabled by default for client-side authentication.
+
+#### CA certificate
+
+Use the [CA certificate](../../../secure/tls-encryption/server-certificates/#generate-the-root-certificate-file) generated above as part of the SSL/TLS configuration of your cluster.
```
Comment (ddhodge) [inline docs/content/preview/drivers-orms/go/pg.md:91]: generated above? you mean, if I happened to follow that link? this need redoing

File: docs/content/preview/drivers-orms/go/pg.md [hunk 20]
```diff
@@ -66,68 +75,288 @@ db := pg.Connect(opt)
...
-## Next steps
+#### Enable SSL/TLS
+
+For a YugabyteDB Managed cluster or a YugabyteDB cluster with SSL/TLS enabled, set the SSL-related environment variables as below.
+
+```sh
+$ export PGSSLMODE=verify-ca
+$ export PGSSLROOTCERT=~/root.crt  # Here, the CA certificate file is downloaded as `root.crt` under home directory. Modify your path accordingly.
+```
+
```
Comment (ddhodge) [inline docs/content/preview/drivers-orms/go/pg.md:344]: ## Run the application

File: docs/content/preview/drivers-orms/orms/go/ysql-gorm.md [hunk 21]
```diff
@@ -26,12 +20,7 @@ type: docs
...
 
-The following tutorial implements an ORM example using [GORM](https://gorm.io/), the ORM library for Golang, that implements a basic REST API server. The scenario is that of an e-commerce application. Database access in this application is managed using GORM. The e-commerce database (`ysql_gorm`) includes the following tables:
-
-- `users` table — the users of the e-commerce site
-- `products` table — the products being sold
-- `orders` table — the orders placed by the users
-- `orderline` table — each line item of an order
+The following tutorial implements an ORM example using [GORM](https://gorm.io/), the ORM library for Golang, that implements a basic REST API server. The scenario is that of an e-commerce application. Database access in this application is managed using GORM.
 
 The source for the above application can be found in the [repository](https://github.com/yugabyte/orm-examples/tree/master/golang/gorm). There are a number of options that can be customized in the properties file located at `src/config/config.json`.
```
Comment (ddhodge) [inline docs/content/preview/drivers-orms/orms/go/ysql-gorm.md:26]: Do Prerequisites, in the format of the C# Dapper example, for all the pages

File: docs/content/preview/drivers-orms/orms/nodejs/ysql-prisma.md [hunk 22]
```diff
@@ -26,12 +26,7 @@ type: docs
...
 
-The following tutorial implements a REST API server using the [Prisma](https://prisma.io) ORM. The scenario is that of an e-commerce application. Database access in this application is managed through the Prisma. It includes the following tables:
-
-- `users` — the users of the e-commerce site
-- `products` — the products being sold
-- `orders` — the orders placed by the users
-- `orderline` — each line item of an order
+The following tutorial implements a REST API server using the [Prisma](https://prisma.io) ORM. The scenario is that of an e-commerce application. Database access in this application is managed through the Prisma.
 
 The source for this application can be found in the [Using ORMs with YugabyteDB](https://github.com/yugabyte/orm-examples/tree/master/node/prisma) repository.
```
Comment (ddhodge) [inline docs/content/preview/drivers-orms/orms/nodejs/ysql-prisma.md:32]: This YB Managed note doesn't link to anything useful - we need a standard boilerplate for YBM that also links to something useful - allow list + certificate. 
All pages need to be updated for YBM and SSL - do it in a separate PR

File: docs/content/preview/drivers-orms/orms/nodejs/ysql-sequelize.md [hunk 23]
```diff
@@ -26,20 +26,15 @@ type: docs
...
+The following tutorial implements a REST API server using the [Sequelize](https://sequelize.org/) ORM. The scenario is that of an e-commerce application. Database access in this application is managed through the Sequelize ORM.
 
 The application source is in the [repository](https://github.com/yugabyte/orm-examples/tree/master/node/sequelize). You can customize a number of options using the properties file located at `config/config.json`.
 
 ## Prerequisites
 
 This tutorial assumes that you have installed:
 
-- YugabyteDB and created a cluster. Refer to [Quick Start](../../../../quick-start/).
+- YugabyteDB and created a cluster. Refer to [Quick Start](../../../quick-start/).
```
Comment (ddhodge) [inline docs/content/preview/drivers-orms/orms/nodejs/ysql-sequelize.md:37]: broken link - check all QS links others are also not right

File: docs/content/preview/drivers-orms/orms/python/ysql-django.md [hunk 24]
```diff
@@ -1,71 +1,58 @@
...
-- `orderline` stores multiple line items from an order.
-
-The source for the above application can be found in the `python/django` directory of Yugabyte's [Using ORMs with YugabyteDB](https://github.com/yugabyte/orm-examples) repository.
-
-## Set up the application
+## Set up the database connection
 
 - Customize the database connection setting according to your environment in the `ybstore/settings.py` file. This file is in the `orm-examples/python/django` directory.
 
 ```python
```
Comment (ddhodge) [inline docs/content/preview/drivers-orms/orms/python/ysql-django.md:51]: indent code blocks under bullets

File: docs/content/preview/integrations/spring-framework/sd-jpa.md [hunk 25]
```diff
@@ -21,7 +19,7 @@ This document describes the fundamentals of using Spring Data JPA with YugabyteD
 
 ## Quick start
 
-Learn how to establish a connection to YugabyteDB database and begin basic CRUD operations using the steps in [Build an Application](../../../develop/build-apps/java/ysql-spring-data/) in the Quick Start section.
+Learn how to establish a connection to YugabyteDB database and begin basic CRUD operations using the steps in [Java ORM example application](../../../drivers-orms/orms/ysql-spring-data/).
```
Comment (ddhodge) [inline docs/content/preview/integrations/spring-framework/sd-jpa.md:22]: bad link

File: docs/content/preview/integrations/spring-framework/sdyb.md [hunk 26]
```diff
@@ -48,8 +48,256 @@ The project definition includes the following dependencies:
 
 ## Examples
 
+### 1. Build a Java application
```
Comment (ddhodge) [inline docs/content/preview/integrations/spring-framework/sdyb.md:51]: What is the app? Make the title about what it does

File: docs/content/preview/integrations/spring-framework/sdyb.md [hunk 27]
```diff
@@ -48,8 +48,256 @@ The project definition includes the following dependencies:
...
 
-The following example demonstrates how to create a basic shopping cart using SDYB:
+The following tutorial illustrates how to build a Java application with Spring Data YugabyteDB and use the YSQL API to connect to and interact with YugabyteDB.
```
Comment (ddhodge) [inline docs/content/preview/integrations/spring-framework/sdyb.md:53]: what is it, and what does it do?

File: docs/content/preview/integrations/spring-framework/sdyb.md [hunk 28]
```diff
@@ -48,8 +48,256 @@ The project definition includes the following dependencies:
...
+You should see the following output:
+
+```output
+2022-04-07 20:25:01.210  INFO 12097 --- [main] com.yugabyte.sdyb.demo.DemoApplication   : Started DemoApplication in 27.09 seconds (JVM running for 27.35)
+Connected to the YugabyteDB server successfully.
+Created table employee
+Query returned: name = User One, email = user@one.com
+```
+
+### 2. Example to create a basic shopping cart
```
Comment (ddhodge) [inline docs/content/preview/integrations/spring-framework/sdyb.md:298]:
```suggestion
### Shopping cart application
```

File: docs/content/preview/integrations/spring-framework/sdyb.md (hunk 27 above)
Comment (aishwarya24) [reply docs/content/preview/integrations/spring-framework/sdyb.md:53]: We are already giving an overview of spring data at the beginning. This example is just a build an app with YB. So I guess the existing details are good.

File: docs/content/preview/drivers-orms/orms/nodejs/ysql-prisma.md (hunk 22 above)
Comment (aishwarya24) [reply docs/content/preview/drivers-orms/orms/nodejs/ysql-prisma.md:32]: okay will create an issue so I remember this. Thanks.
